plugins/woocommerce-blocks/tests/e2e/README.md

```diff
@@ -142,3 +151,14 @@ To see all options, run the following command:
 ```sh
 npx playwright test --help
 ```
+
+### Troubleshooting
+
+If you run into problems the first time you try to run the tests, please run the following command before starting the test suite:
+
+```sh
+pnpm wp-env:config
+```
+
+This helps set up your environment correctly and can prevent some of the usual
+issues from happening.
```

Comment:
Is there a reason to split this in two lines?

```suggestion
This helps set up your environment correctly and can prevent some of the usual issues from happening.
```

Reply:
Good catch! That must have been the [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) plugin wrapping automatically at >80 cols. I need to disable it for the *.md files 👀

Reply:
I've just noticed that the wrapping is generally [inconsistent in that file](https://raw.githubusercontent.com/woocommerce/woocommerce/3e21a9d5a6b19734a0820139ce811c3ef044eb5b/plugins/woocommerce-blocks/tests/e2e/README.md). Let me know if it should be corrected one way or another. Personally, I like things wrapped in *.md files since it's easier to read in most cases.

Reply:
Honestly, I don't know what's the way to go and I don't think it's documented anywhere. However, I took a look at some other files (ie: [file 1](https://raw.githubusercontent.com/woocommerce/woocommerce/trunk/docs/wc-cli/cli-faq.md), [file 2](https://raw.githubusercontent.com/woocommerce/woocommerce/trunk/docs/contributing-docs/style-guide.md), [file 3](https://raw.githubusercontent.com/woocommerce/woocommerce/trunk/docs/data-management/crud-objects.md)) and it looks like WC core doesn't usually wrap *.md files? :thinking:

Reply:
Yep, looks like unwrapped would be the way to go! 😄 OK if I unwrap the rest of the file within this PR?

Reply:
> OK if I unwrap the rest of the file within this PR?

Sure thing! :+1:

Reply:
...and the lines have been unwrapped! 🎉 Let me know if it looks good. 😄

Reply:
Yes, LGTM. :slightly_smiling_face: